.github/workflows/analysis.yml

```diff
@@ -12,39 +12,36 @@ on:
 
 env:
   build: '${{ github.workspace }}/build'
-  results: '${{ github.workspace }}/build/results'
+  config: 'Debug'
 
```

Comment:
Only `Debug` needs analysis?  Wouldn't use to want to be interested in 'retail'?

Reply:
Selection of Check/Debug over Release comes down to whether or not you want the analysis engine to understand assertion or ignore the assumptions they make. There is trade-offs/benefits to either approach so I will let you decide which is appropriate for this repo.

Reply:
I would like to encourage analysis of the retail code over check/debug for this repo.